src/components/GameRound.vue

```diff
@@ -70,11 +84,15 @@ const checkSomePlayersHaveNoGabo = () => {
     <ButtonRedesigned @click="handleEndRoundClick" class="end-round">
       Prochaine manche
     </ButtonRedesigned>
+    <ButtonRedesigned variant="ghost" @click="handleResetGame" class="mt-2">
+      Reset game
+    </ButtonRedesigned>
     <div v-if="allPlayersHaveZeroPoints || somePlayersHaveNoGabo" class="issue">
       Vous devez indiquer le nombre de points de chaque joueur et désigner qui a
       prononcé Gabo
     </div>
   </div>
+  <ResetGameConfirmView @onClose="closeMenu" :isOpen="!isResetGameModalHidden"/>
```

Comment:
I'm not sure "View" is the right word for a modal. Maybe it could be a reusable modal component called `Modal` and you pass the children as a content of your modal?